README.md

```diff
@@ -979,6 +979,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 * [bluss/ndarray](https://github.com/rust-ndarray/ndarray) — N-dimensional array with array views, multidimensional slicing, and efficient operations
 * [kernelmachine/utah](https://github.com/kernelmachine/utah) — Dataframe structure and operations in Rust
+* [ritchie46/polars/](https://github.com/ritchie46/polars) - Fast feature complete DataFrame library
```

Comment:
Please add the Travis build badge and lose the "/" from the end of the name bit. Thanks!

Reply:
Done!